Remove experimental status tag and references to experimental status on website (#3198)
alphagov/govuk-design-system: src/community/contribution-criteria/index.md

      },
      {
        html: '<p>It has been tested  in user research and shown to work with a representative sample of users, including those with disabilities.</p>

>           <p class="govuk-!-margin-bottom-0">Components and patterns published in a limited number of services can still be published, as long as it meets other contribution criteria. We\'ll be clear about the specific contexts the component or pattern has been tested in and encourage feedback about its performance in a wide range of services.</p>'

Reviewer: This line read a bit funny that published patterns could still be published?

```suggestion
          <p class="govuk-!-margin-bottom-0">Components and patterns published in a limited number of services can still be accepted, as long as it meets other contribution criteria. We’ll be clear about the specific contexts the component or pattern has been tested in and encourage feedback about its performance in a wide range of services.</p>'
```

Reviewer 2: Thanks Colin, I might go with 'contributed' instead :)

Reviewer: Fine by me. Press that merge button 🎉

>           <p class="govuk-!-margin-bottom-0">If there isn\'t evidence of usability outside of one or a small number of services, the component or pattern can still be published if it meets other contribution criteria. We will be clear about what contexts the component or pattern has been tested in upon publishing and invite feedback about its performance in a wide range of services.</p>'

Reviewer 2: ```suggestion
          <p class="govuk-!-margin-bottom-0">If there is not evidence of usability outside of one or a small number of services, the component or pattern can still be published if it meets other contribution criteria. We will be clear about what contexts the component or pattern has been tested in upon publishing and invite feedback about its performance in a wide range of services.</p>'
```

I wonder if @calvin-lau-sig7 can give this a once over?

Reviewer 3: Once over'ed! How's this look?

```suggestion
          <p class="govuk-!-margin-bottom-0">Components and patterns published in a limited number of services can still be published, as long as it meets other contribution criteria. We'll be clear about the specific contexts the component or pattern has been tested in and encourage feedback about its performance in a wide range of services.</p>'        
```

Reviewer 2: @owenatgov what do you think? Should I commit this change?
